src/main/java/core/basesyntax/security/AuthenticationServiceImpl.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.security;
+
+import core.basesyntax.exception.AuthenticationException;
+import core.basesyntax.lib.Inject;
+import core.basesyntax.lib.Service;
+import core.basesyntax.model.Driver;
+import core.basesyntax.service.DriverService;
+
+@Service
+public class AuthenticationServiceImpl implements AuthenticationService {
+    @Inject
+    private DriverService driverService;
+
+    @Override
+    public Driver login(String login, String password) throws AuthenticationException {
+        Driver driver = driverService.findByLogin(login)
+                .orElseThrow(() -> new AuthenticationException("Incorrect login. "
```

Comment:
with this message, it's easier to get access to the user account (first you're guessing login and you will know for sure what accounts exist) let's use the same message in both cases

Reply:
ok, I got a point)